src/Nethermind/Nethermind.Consensus/Messages/BlockErrorMessages.cs

```diff
@@ -116,4 +117,7 @@ public static class BlockErrorMessages
 
     public const string NegativeGasUsed =
         "NegativeGasUsed: Cannot be negative.";
+    public static string MissingDeposits => "Deposits cannot be null in block when EIP-6110 activated.";
```

Comment:
The other error messages starts with a machine parseable "code". These messages should do the same.

```suggestion
    public static string MissingDeposits => "MissingDeposits: Deposits cannot be null in block when EIP-6110 activated.";
```